org.eclipse.wildwebdeveloper/schema/org.eclipse.wildwebdeveloper.jsonschemaassociations.exsd

```diff
@@ -0,0 +1,238 @@
+&lt;/ul&gt;
+&lt;/br&gt;
+* If it becomes necessary to retrieve the file and keep the bundle as a jar, the implementation of the file locating needs to be adjusted and support a bundle directory.
+&lt;/p&gt;
+      </documentation>
+   </annotation>
+
+   <element name="extension">
+      <annotation>
+         <appinfo>
+            <meta.element />
+         </appinfo>
+      </annotation>
+      <complexType>
+         <sequence>
+            <element ref="schemaAssociations" minOccurs="1" maxOccurs="unbounded"/>
+         </sequence>
+         <attribute name="point" type="string" use="required">
+            <annotation>
+               <documentation>
+                  The identifier of the target extension point
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="id" type="string">
+            <annotation>
+               <documentation>
+                  An optional identifier of the extension instance
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="name" type="string">
+            <annotation>
+               <documentation>
+                  An optional name of the extension instance
+               </documentation>
+               <appinfo>
+                  <meta.attribute translatable="true"/>
+               </appinfo>
+            </annotation>
+         </attribute>
+      </complexType>
+   </element>
+
+   <element name="schemaAssociations">
+      <annotation>
+         <documentation>
+            Sequence of all JSON schema associations
+         </documentation>
+      </annotation>
+      <complexType>
+         <sequence minOccurs="1" maxOccurs="unbounded">
+            <sequence minOccurs="1" maxOccurs="unbounded">
+               <choice>
+                  <element ref="bundleFilePath"/>
+                  <element ref="url"/>
+               </choice>
+            </sequence>
+            <sequence minOccurs="1" maxOccurs="unbounded">
+               <element ref="filePattern"/>
+            </sequence>
+         </sequence>
+         <attribute name="id" type="string" use="required">
+            <annotation>
+               <documentation>
+                  Title of schema association
+               </documentation>
+            </annotation>
+         </attribute>
+      </complexType>
+   </element>
+
+   <element name="bundleFilePath">
+      <complexType>
+         <attribute name="relativePath" type="string" use="required">
+            <annotation>
+               <documentation>
+                  The relative path of the JSON schema file within the bundle (i.e. the plug-in).
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="bundleId" type="string">
+            <annotation>
+               <documentation>
+                  An optional symbolic ID of the bundle (i.e. the plug-in ID).
```

Comment:
Do you have a use-case for it? If not, you could remove this field.
If you have a use case for it, please consider merging it with the `relativePath` entry, so if it's a path, it will load a resource in the bundle; but it can also allow resource URLs like `platform:/plugin/some.other.bundle/path/to/file.schema.json`. I just found https://lmap.blogspot.com/2008/03/platform-scheme-uri.html which gives similar example for icon loading.